src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -227,6 +227,14 @@ MUST be present **in this specific order**:
 | type            | REQUIRED              | Type of channel; MUST use the channel types listed below. Note that the type MUST be in upper-case.                                                                           |
 | units           | REQUIRED              | Physical unit of the value represented in this channel, for example, `V` for Volt, or `fT/cm` for femto Tesla per centimeter (see [Units](../02-common-principles.md#units)). |
 
+{{ MACROS___make_columns_table(
+   {
+      "name_channels": "REQUIRED",
```

Comment:
this now contains iEEG specific info, which seems out of place in the MEG spec :thinking: 

> name and all ieeg type channels are required to have a position. The reference channel name MAY be provided in the reference column

Can we have this info only in the iEEG channel name column description?

Reply:
Just to be sure, does the additional info not apply to MEG/EEG columns at all, or is the problem just that the term `ieeg` is used in the definition? If it's the former then I can definitely just append this info to the description within the table like we do with metadata fields. If it's the latter then I'd prefer to generalize the description.